### ISSUE TITLE
Can't Install on Mac: libxerces-c-3.1.dylib Problem

### ISSUE THREAD

**Issue by astrostu** [DOI-USGS/ISIS3#4637]
**ISIS version(s) affected**: 5.0.2, 6.0.0

**Description**  
Installed ISIS 6.0.0 via the conda solver (mamba not working, that's a separate issue on here), and it won't run.  This is on a Mac running macOS 10.15.7, Intel CPU.
```
dyld: Library not loaded: @rpath/libssh2.1.dylib
  Referenced from: /Users/stu/anaconda3/envs/isis6/lib/libxerces-c-3.1.dylib
  Reason: image not found
Abort trap: 6
```

**How to reproduce**  
I created a new conda environment (with Python 3.6), loaded the channels, installed, then tried to run anything and got the above error.  I'll note, if it's relevant, that there was no install of libssh* when I did the ISIS install.

**Possible Solution**  
Something is fixed on your end?

**Additional context**  
None noted.

**Comment by jlaura** [comment]
This could be related to packages already in your env that need to be cleaned. 

I will also note that we do not have access to 10.15 machines and therefore do not support that OS. Should it matter for conda? Not that I am aware of. Are the ASC ISIS developers able to test and assert that assumption at this time? No.

Is this reproducible on a 10.13 or 10.14 OS?

**Comment by astrostu** [comment]
@jlaura I have no idea if it's replicable on earlier OSs, I only have machines running 10.15 and 11.6 [for those reading this, Apple's yearly releases were 10.X since they redid their OS to be Unix-based, then in 2020, the #.X updates were their minor/security updates so macOS 11.6 is still the variation on their 2020 major OS update].

I will note that I finally got conda to solve for 5.0.2, and I get the same error.  To be very clear, these are completely, 100% new conda environments I made, the only thing installed is ISIS.  I've gone back and updated the versions affected in the first post.

BTW, the DOI won't let you folks update to an operating system released in 2019??  MacOS 10.15 has been out since ≈September 2019, it's practically 2 years old now.

**Comment by jlaura** [comment]
When you say a clean environment it is ambiguous if you have other environments in your conda area. The solvers can solve using local packages in other environments. Checkout `conda clean` and let us know if a clean fixes your issue.

Also, this is not an appropriate place to provide editorial opinions about organizational decisions regarding our computer update cadence. If that is a concern for you, please email me directly: jlaura@usgs.gov.
